### PR TITLE
gem update

### PR DESCRIPTION
**Pull request by 0xTheProDev**

### CODE REVIEW

File: _site/projects/index.html [hunk 1]
```diff
@@ -151,8 +151,8 @@ <h1 class="page-title">Projects</h1>
 	<div class="browse-medium">
 	
 	
-	<a href="http://bytesclub.github.io/assets/tictactoe/">
-		<img src="http://bytesclub.github.io/img/assets/tictactoe/OneThird.jpg" alt="tictactoe" />
+	<a href="http://progyan1997.github.io/BytesClub.github.io/assets/tictactoe/">
```
Comment (RudraNilBasu) [inline _site/projects/index.html]: Why is this changed?

File: _site/projects/index.html [hunk 2]
```diff
@@ -151,8 +151,8 @@ <h1 class="page-title">Projects</h1>
...
+		<img src="http://progyan1997.github.io/BytesClub.github.io/img/assets/tictactoe/OneThird.jpg" alt="tictactoe" />
```
Comment (RudraNilBasu) [inline _site/projects/index.html]: Same here (and in lines: 176, 177, 190, 191, 212, 213)

File: _site/atom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
  <title>BytesClub</title>
  <link href="https://bytesclub.github.io//atom.xml" rel="self"/>
  <link href="https://bytesclub.github.io//"/>
- <updated>2017-06-03T11:27:42+05:30</updated>
+ <updated>2017-06-19T12:09:51+05:30</updated>
```
Comment (RudraNilBasu) [inline _site/atom.xml]: Why change it here?

Comment (0xTheProDev) [reply _site/atom.xml]: Because  opened it.

**Comment by RudraNilBasu** [comment]
There is a problem with the project pages here:

screenshot from 2017-06-20 22-12-15

screenshot from 2017-06-20 22-12-18
